src/axom/quest/detail/shaping/WindingNumberSampler.hpp

```diff
@@ -81,10 +81,13 @@ class WindingNumberSampler
    * \param geomView A view that contains the shapes being queried.
    *
    */
-  WindingNumberSampler(const std::string& shapeName, GeometryView geomView)
-    : m_shapeName(shapeName)
-    , m_geometryView(geomView)
-  { }
+  WindingNumberSampler(const std::string& shapeName, GeometryView geomView) : m_shapeName(shapeName)
+  {
+    for(auto & contour : geomView)
```

Comment:
Minor:

```suggestion
    for(const auto & contour : geomView)
```

Assuming that caching doesn't change the contour, and mostly to check if all the required functions are marked `const`